WebApiCoreSeed.UnitTests/Infrastructure/RestClient/TestDto.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace WebApiCoreSeed.UnitTests.Infrastructure.RestClient
+{
+    public class TestDto

```

Comment:
The only _real_ use of this DTO is on test `PostAsync_ShouldReturnResourceObject_WhenRequestIsSuccessful`, that's the only test where you're actually parsing and using it.

The name is way too generic and I prefer not to share such a class between test methods. When you don't care about success response (i.e. in any other use of this class besides the test I mentioned), just use `object ` or something else.